lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/metrics/TDigestHistogram.java

```diff
@@ -100,66 +104,77 @@ public synchronized long getTotal()
     // Below are extra properties that make it easy to read and parse serialized distribution
     // in operator summaries and event listener.
     @JsonProperty
+    @JsonSerialize(using = DoubleFormatter.class)
     public synchronized double getMin()
     {
         return digest.getMin();
     }
 
     @JsonProperty
+    @JsonSerialize(using = DoubleFormatter.class)
```

Comment:
This won’t work if the class is going through the SPI, since only Jackson annotations are part of the SPI.

Reply:
At the moment all the usages seem to be in trino-main, probably that's why tests didn't fail. But the class is in plugin-toolkit, so it can be used by some connector. 
Is there a safe way to achieve the same end result ? I'm trying to have these values printed in the queryinfo json in a shorter and easier to read way rather than with too many decimal places.

Reply:
I've removed this change